specification/src/main/asciidoc/platform/RelatedDocuments.adoc

```diff
@@ -25,7 +25,7 @@ _Java™ Naming and Directory Interface 1.2 Specification (JNDI specification)_.
 
 _Jakarta™ Messaging Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/messaging/2.0_
 
-_Jakarta™ Transaction Specification, Version 1.2_. Available at: _https://jakarta.ee/specifications/transactions/1.3_
+_Jakarta™ Transaction Specification, Version 1.3_. Available at: _https://jakarta.ee/specifications/transactions/1.3_
```

Comment:
Thanks for doing this, the project in this case is Jakarta Transactions [1] actually

[1] https://projects.eclipse.org/projects/ee4j.jta

Reply:
Nice catch, @tomjenkinson!  Thanks.